### PR TITLE
Task/export fill service

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,2 @@
-- Fix: keep internalAttributes stored in device mongo collection when update it (#1092)
+- Export fillDomains function to iotagent-node-lib available functions
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Is there any issue associated?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: No issue related

**Comment by AlvaroVega** [comment]
Consumed by https://github.com/telefonicaid/iotagent-ul/pull/512

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**